yield_plugin/src/yield_plugin.cpp

```diff
@@ -103,15 +104,17 @@ namespace yield_plugin
       // roadway object position
       double gap_time = (x_lead - config_.x_gap)/current_speed_;
 
-      double collision_time = 0; //\TODO comming from carma_wm collision detection in future (not used now)
+      double collision_time = 0; //\TODO comming from carma_wm collision detection in future (CAR 4288)
 
       double goal_velocity = rwol_collision[0].object.velocity.twist.linear.x;
       
+      double goal_pos = x_lead - goal_velocity * gap_time; 
```

Comment:
Might be paranoid, but it might not be a bad idea to ensure that the second term here is always > x_gap with a std::min() call. The way we have it now isn't a problem with min_obstacle_speed being 1.0 by default, but if that value were to get tweaked for whatever reason to something less than 1.0, then goal_velocity * gap_time could be less than x_gap.